pygeos/strtree.py

```diff
@@ -49,7 +49,8 @@ class STRtree:
     """
 
     def __init__(self, geometries, leafsize=5):
-        self._tree = lib.STRtree(np.asarray(geometries, dtype=np.object), leafsize)
+        self.geometries = np.asarray(geometries, dtype=np.object)
```

Comment:
@caspervdw I'm bumping into issues merging this into #108 

```
  File "/Users/bcward/projects/pygeos/pygeos/strtree.py", line 52, in __init__
    self.geometries = np.asarray(geometries, dtype=np.object)
AttributeError: can't set attribute
```

This is conflicting with the `geometries` attribute.  How did you get past this?

Reply:
Oh - whoops!  Merge issue on my end!  Please ignore.